integer/src/convert.rs

```diff
@@ -259,6 +259,62 @@ impl IBig {
             Inexact(val, diff) => Inexact(sign * val, sign * diff),
         }
     }
+
```

Comment:
The intended way to convert big integers into small integers is through the `TryFrom` APIs, there is no need to create standalone functions. (besides, convert to floats first is very inefficient).

To do a narrowing conversion, you can do: `let small: i32 = big.try_into().unwrap()`